index.html

```diff
@@ -0,0 +1,461 @@
+                    <path d="M94.5324 13.1308L93.1657 19.0488C93.1234 19.228 93.1361 19.4158 93.2021 19.5877C93.2682 19.7596 93.3846 19.9075 93.536 20.0122C93.6875 20.1169 93.867 20.1735 94.0511 20.1746C94.2353 20.1757 94.4154 20.1212 94.5681 20.0183L99.5625 16.6888L104.557 20.0183C104.713 20.1221 104.897 20.1755 105.085 20.1714C105.272 20.1673 105.454 20.1058 105.606 19.9953C105.757 19.8849 105.871 19.7306 105.932 19.5534C105.994 19.3761 105.999 19.1844 105.948 19.0039L104.271 13.1336L108.431 9.38944C108.564 9.26946 108.66 9.11304 108.705 8.93955C108.75 8.76605 108.744 8.58308 108.686 8.41327C108.629 8.24345 108.522 8.09424 108.381 7.98408C108.239 7.87392 108.069 7.80764 107.89 7.79345L102.661 7.37703L100.398 2.36801C100.326 2.20672 100.209 2.06976 100.06 1.97366C99.9121 1.87755 99.7392 1.82642 99.5625 1.82642C99.3858 1.82642 99.2129 1.87755 99.0646 1.97366C98.9163 2.06976 98.799 2.20672 98.7269 2.36801L96.4641 7.37703L91.2349 7.79254C91.0592 7.80645 90.8913 7.87069 90.7511 7.97757C90.611 8.08445 90.5046 8.22943 90.4447 8.39518C90.3848 8.56094 90.3739 8.74044 90.4134 8.91221C90.4528 9.08398 90.5409 9.24074 90.6671 9.36376L94.5324 13.1308ZM97.1492 9.16288C97.313 9.14998 97.4702 9.09328 97.6045 8.99871C97.7387 8.90414 97.8451 8.77519 97.9124 8.62538L99.5625 4.97387L101.213 8.62538C101.28 8.77519 101.386 8.90414 101.52 8.99871C101.655 9.09328 101.812 9.14998 101.976 9.16288L105.619 9.45181L102.619 12.1521C102.358 12.387 102.255 12.7484 102.35 13.0859L103.499 17.108L100.072 14.8231C99.922 14.7222 99.745 14.6683 99.5638 14.6683C99.3827 14.6683 99.2057 14.7222 99.0552 14.8231L95.4744 17.2107L96.4375 13.0409C96.4728 12.8876 96.468 12.7277 96.4237 12.5767C96.3794 12.4257 96.2969 12.2887 96.1843 12.1787L93.3977 9.4619L97.1492 9.16288Z" fill="#F3CD03"/>
+                    </g>
+                    <defs>
+                    <clipPath id="clip0_964_6448">
+                    <rect width="110.068" height="22" fill="white" transform="translate(0.5)"/>
+                    </clipPath>
+                    </defs>
+                    </svg>
+                <p class="card__desc">
+                    Slate helps you see how many <br/>
+                    more days you need to work to <br/>
+                    reach your financial goal.
+                </p>    
+                <div class="card__header">
+                    <div class="card__img">
+                        <img src="./img/cover (1).png"  class="card__cover" />                       
+                    </div>
+                    <div class="card__name">
+                        <p class="card__p card__p--active">Regina Miles</p>
+                        <p class="card__p">Designer</p>
+                    </div>
```

Comment:
### warning name not explicite

`card__p` is not explicite it will be good if you have a good name like `card__paragraph`